tests/cql/CqlTypesTest.xml

```diff
@@ -74,11 +74,11 @@
 		</test>
 		<test name="DateTimeIncomplete">
 			<expression>DateTime(2015, 2, 10)</expression>
-			<output>@2015-02-10</output>
+			<output>@2015-02-10T</output>
 		</test>
 		<test name="DateTimeUncertain">
 			<expression>days between DateTime(2015, 2, 10) and DateTime(2015, 3)</expression>
-			<output>Interval [ 19, 49 ]</output>
+			<output>Interval [ 18, 49 ]</output>
```

Comment:
* lower: (whole) days between @2015-02-10T23:59:59.999 and @2015-03-01T00:00:00.000 =`18`
* upper: (whole) days between @2015-02-10T00:00:00.000 and @2015-03-31T23:59:59.999 = `49`